dicom-mr-classifier.py

```diff
@@ -227,8 +227,10 @@ def get_sex_string(sex_str):
         sex = "male"
     elif sex_str == "F":
         sex = "female"
+    elif sex_str == "O":
+        sex = "other"
     else:
-        sex = ""
+        sex = "unknown"
```

Comment:
Does the field have to be populated (i.e., "unknown")? Is there anything that will be messed up by returning a non-empty value?

Reply:
It doesn't need to be populated, but `unknown` is an allowed field, so I figured it would be better to have that, I can definitely change it to empty if we want.